code/modules/mob/living/carbon/human/human.dm

```diff
@@ -936,6 +937,17 @@
 			var/msg = span_notice("[key_name_admin(usr)] has removed [key_name(src)] from purrbation.")
 			message_admins(msg)
 			admin_ticket_log(src, msg)
+	//SKYRAT EDIT ADDITION BEGIN - CRYO SEND
+	if(href_list[VV_HK_SEND_CRYO])
+		if(!check_rights(R_SPAWN))
+			return
+		src.send_to_cryo()
+		to_chat(usr, "Put [src] in cryopod.")
+		log_admin("[key_name(usr)] has put [key_name(src)] to cryopod.")
```

Comment:
I'd like it if you could add the location to this log as well.

Also, the content of the log itself should be stored into a variable instead, so that you can add the `span_notice()` around it only for the `message_admins()` portion.
```suggestion
		log_admin("[key_name(usr)] has put [key_name(src)] into a cryopod.")
```